test/helpers/logger.go

```diff
@@ -25,6 +25,10 @@ func (l Logger) Section(msg string, f func()) {
 	f()
 }
 
+func (l Logger) Warnf(msg string, args ...interface{}) {
```

Comment:
do we need warnings in tests?

Reply:
no, but we need to comply with `util.LoggerWithLevels` interface